### PR TITLE
move subpath.md to pv.md

### PR DESCRIPTION
**Pull request by timfeirg**
* delete subPath related content, as it's not really recommended
* move the rest content from subpath.md to pv.md
* make mountOptions related content a separate section
* rename "Create mount configuration" to "File system credentials" (pending translation)

### CODE REVIEW

File: docs/zh_cn/guide/subpath.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  persistentVolumeReclaimPolicy: Retain
+  csi:
+    driver: csi.juicefs.com
+    volumeHandle: juicefs-pv
+    fsType: juicefs
+    nodePublishSecretRef:
+      name: juicefs-secret
+      namespace: default
+    volumeAttributes:
+      subPath: fluentd
```
Comment (timfeirg) [inline docs/zh_cn/guide/subpath.md]: so, a directory named fluentd will be created under the JuiceFS pv directory? @zwwhdls

File: docs/zh_cn/guide/subpath.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    nodePublishSecretRef:
+      name: juicefs-secret
+      namespace: default
+  mountOptions:
+    - subdir=/test
+```
+
+## 使用 `subPath`
+
+在 PV 中这样使用 `subPath`，如果指定的子目录不存在，CSI Controller 会自动创建。
```
Comment (timfeirg) [inline docs/zh_cn/guide/subpath.md]: who will be in charge of creating the subPath directory? CSI Controller? or some other components? @zwwhdls

Comment (zwwhdls) [reply docs/zh_cn/guide/subpath.md]: Mistake, CSI Node will create it.

File: docs/zh_cn/guide/subpath.md (hunk 1 above)
Comment (zwwhdls) [reply docs/zh_cn/guide/subpath.md]: Yes, CSI Node will create it and bind it to target in application pod.

**Comment by timfeirg** [comment]
subPath may not be useful to users, consider removing it from docs

**Comment by timfeirg** [comment]
translation done @xiaogaozi

**Review by xiaogaozi — APPROVED** [review]

**Pull request merged**